templates/todo/projects/nodejs-mongo/repo-bicep.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   description: ToDo Application with a Node.js API and Azure Cosmos DB API for MongoDB
 
 repo:
-  includeProjectAssets: true
+  includeProjectAssets: false
```

Comment:
What switch to false?

Reply:
in this proposed design, Infra folder can have multi IAC tools code, we need to be more specific about what gets copied. this is being set to false and a new rule added to replace its effect that ensure to copy all except infra,  https://github.com/Azure/azure-dev/blob/ec30798409a7fa0bc7cf234ef2c1933516b69c89/templates/todo/projects/nodejs-mongo/repo-bicep.yaml#L21

with explicit rules on how the infra content is being generated . https://github.com/Azure/azure-dev/blob/ec30798409a7fa0bc7cf234ef2c1933516b69c89/templates/todo/projects/nodejs-mongo/repo-bicep.yaml#L73